src/content/contributing/adding-layer-2/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Adding Layer 2's
```

Comment:
Grammar nitpick, should it be "Layer 2's"? Or "Layer 2s"? What do you say @minimalsm 😄?